tutorials/README.md

```diff
@@ -0,0 +1,7 @@
+# Enketo Core tutorials
+
+* [Getting started](getting-started.md)
+* [Configuration](configuration.md)
+* [Development](development.md)
+* [Events](events.md)
+* [Creating/extending widgets](widgets.md)
```

Comment:
Why not just let GitHub provide the navigation?

Reply:
I... don't know?